the-collab-lab/tcl-65-smart-shopping-list
# sb layout design

## Description
This PR adds UI styling. Changes made include:

- Added four background images to _resources folder
- Text, input fields and `Buttons` were centered throughout the app
- `Buttons` were resized
- Background image was added to the app
- Light mode and dark mode images added 

I installed:
- Material UI https://mui.com/material-ui/react-button/
- Styled Components https://styled-components.com/docs

But decided in the end to stick to plain CSS to make these changes.


## Related Issue
Closes #13 

## Acceptance Criteria
- [X] Resize `Buttons` throughout app
- [X] Center Layout and add spacing between text and `Buttons` throughout app
- [X] Custom design artwork for Dark Mode and Light Mode  background images with two variations
- [X] Add background images to Light Mode and Dark Mode media query Light Mode set as default

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

https://github.com/the-collab-lab/tcl-65-smart-shopping-list/assets/86745260/833416e6-fff0-4cb9-b88b-e78919ea5bb7




### After

https://github.com/the-collab-lab/tcl-65-smart-shopping-list/assets/86745260/3675e506-109d-4538-bbe8-8a8ab043f59b




## Testing Steps / QA Criteria

- Use GitHub Preview
- Navigate to all pages and checkout UI changes, hover over `Buttons`
- Change your browser from Light Mode to Dark Mode to see changes
